L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer2EG_cff.py

```diff
@@ -49,6 +52,22 @@
         nTKELE_OUT=cms.uint32(12),
         nTKPHO_OUT=cms.uint32(12),
     ),
+    egPFIso = cms.PSet(
```

Comment:
So `dZMax` should become `dZ`?

Reply:
yes. But more importantly, for clarity, I would rename the whole PSet: otherwise, the user will find 2 "pfIsolation" parameter sets and might get confused.
For consistency, I would propose to use something like:
`puppiIsoParametersTkEle` and `puppiIsoParametersTkEm`